src/main/java/Buddy.java

```diff
@@ -0,0 +1,158 @@
+import java.util.Scanner;
+
+public class Buddy {
+    public static void main(String[] args) {
```

Comment:
Your main method seems to be quite long as mentioned by the other peer reviewer. You could probably consider extracting each of the code blocks under the conditionals into separate methods ? That should create an additional layer of abstraction and increase the readability of your code !